.github/CODEOWNERS

```diff
@@ -3,9 +3,6 @@
 
 *                                        @TooTallNate @EndangeredMassa @styfle @cb1kenobi @Ethan-Arrowood
 /.github/workflows                       @TooTallNate @EndangeredMassa @styfle @cb1kenobi @Ethan-Arrowood @ijjk
-/packages/cli/src/commands/domains       @mglagola @anatrajkovska
-/packages/cli/src/commands/certs         @mglagola @anatrajkovska
-/packages/cli/src/commands/env           @TooTallNate @EndangeredMassa @styfle @cb1kenobi @Ethan-Arrowood
```

Comment:
I think `env` should have @baruchadi and the team working on env vars right now.

Reply:
I added them as a reviewer and reached out on Slack to confirm

Reply:
I asked around and didn't get a response yet. I say we merge this and update it if they get back to me.